examples/protein-ligand/cli/protein-ligand.yaml

```diff
@@ -67,9 +67,13 @@ atom_selection: not water
 phases:
     - complex
     - solvent
-    - vacuum
+#    - vacuum
 
 #timestep in fs
 timestep: 4.
 
 h_constraints: True
+
+offline-freq: 2
```

Comment:
Can you add some comments on what these do? Like is the ofline-freq in steps? time?

Reply:
Yes!